apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConversionOptions.cs

```diff
@@ -174,12 +158,6 @@ private void ValidateAndParseSpannerToClrTypeMappings(string input, SpannerConve
             }
 
             // Check multiples for each type.
-            // Currently, we have Float64 and Date to check.
```

Comment:
I'm not checking this part massively carefully as I still hope to be able to refactor it. Although there's a bit less need now...